benchmark/main.py

```diff
@@ -1,15 +1,19 @@
 import sys
-CPYTHON = not hasattr(sys, 'pypy_version_info')
 import time
 import json
 import ujson
 
-from tests.support import import_ujson_universal
-ujson_hpy_universal = import_ujson_universal()
+from tests.support import import_ujson_hpy
+
+ujson_hpy_universal = import_ujson_hpy("universal")
 ujson_hpy_universal.__name__ = 'ujson_hpy_universal'
 
+ujson_hpy_debug = import_ujson_hpy("debug")
+ujson_hpy_debug.__name__ = 'ujson_hpy_debug'
+
+CPYTHON = not hasattr(sys, 'pypy_version_info')
```

Comment:
Since this is very specific to PyPy, could we use `CPYTHON = sys.implementation.name == "cpython"` ?